aggregator/src/lib.rs

```diff
@@ -12,6 +12,12 @@ pub mod diagnostic;
 pub mod metrics;
 pub mod trace;
 
+#[cfg(test)]
+extern crate quickcheck;
```

Comment:
`extern crate` is pretty archaic; can we `use quickcheck` instead?

Reply:
Ah, we totally can. I copypasta'd from the examples without being critical about it.